services/web/client/source/class/osparc/data/model/Workbench.js

```diff
@@ -194,10 +196,10 @@ qx.Class.define("osparc.data.model.Workbench", {
       node.giveUniqueName();
 
       // create the node in the backend here
-      const prjId = this.getStudy().getUuid();
+      const study = osparc.store.Store.getInstance().getCurrentStudy();
```

Comment:
so we are storing the full study instead of just the id, this leads to data duplicates right? in ``currentStudy`` and ``studies``... are you dealing with syncronizing both copies? I sense problems coming with this approach

Reply:
AFAIK, there is only one Study model and all instances use a reference to it.